lib/saxerator.rb

```diff
@@ -4,6 +4,9 @@
 require 'saxerator/dsl'
 require 'saxerator/full_document'
 require 'saxerator/document_fragment'
+
+require 'saxerator/adapters/nokogiri'
```

Comment:
We can't require these without making both `nokogiri` and `ox` dependencies. The idea of excluding them here and requiring them inside the config is to make them optional so you can choose whatever XML library you want.